crud/select/comparators.lua

```diff
@@ -158,23 +169,29 @@ local array_cmp_funcs_by_operators = {
     [operators.GE] = array_ge,
 }
 
-function comparators.gen_func(operator, key_parts)
-    local cmp_func = array_cmp_funcs_by_operators[operator]
+function comparators.get_cmp_operator(tarantool_iter)
+    local cmp_operator = cmp_operators_by_tarantool_iter[tarantool_iter]
+    assert(cmp_operator ~= nil, 'Unknown Tarantool iterator %q', tarantool_iter)
```

Comment:
Unsupported or unknown

```
tarantool> box.index
---
- EQ: 0
  GT: 6
  BITS_ALL_SET: 7
  GE: 5
  NEIGHBOR: 11
  OVERLAPS: 10
  REQ: 1
  BITS_ALL_NOT_SET: 9
  BITS_ANY_SET: 8
  LT: 3
  LE: 4
  ALL: 2
...
```

There are some iterator types for bitset/rtree indexes

Reply:
Unknown, you are right.